qa/tasks/deepsea_deploy.py

```diff
@@ -716,6 +724,47 @@ def _run_stage_2(self, config):
         self._dump_global_conf()
         self._dump_mon_conf()
 
+    def _custom_ceph_conf(self, config):
+        """
+        Add custom options for ceph.conf
+        Dict definition should be defined between stage 2 and 3.
+        Conf files for each roles are stored in:
+        /srv/salt/ceph/configuration/files/ceph.conf.d/
+        EXAMPLE:
+        tasks:
+          - deepsea_deploy:
+              cli: true
+              commands:
+                - stage0:
+                    update: false
+                - stage1:
+                - policy_cfg:
+                    profile: default
+                - stage2:
+                - custom_ceph_conf:
+                    global:
+                      mon lease: 15
+                      mon lease ack timeout: 25
+                    mon:
+                      debug mon: 20
+                    osd:
+                      debug filestore: 20
+                - stage3:
+        """
+        log.info("deepsea_deploy: WWWW: custom options in ceph.conf")
+        if not config:
+            info_msg = ("deepsea_deploy: empty config ")
+            log.info(info_msg)
```

Comment:
The `deepsea_deploy: ` prefix is not necessary when using `log.info` (as opposed to `self.log.info`).

Also, the `info_msg` variable is not needed here - this could be changed to simply:

```
if not config:
    log.info("empty config: not setting any custom ceph.conf")
    return None
```

Reply:
I don't know which kind of problem do you resolve. I think it is overkill to prefix logging with function name, if you really need this kind of information it is best to change the template for logger.

Reply:
@kshtsk This was my mistake. When using `log.info()` the prefix is not needed. I had added the prefix because `self.log.info()` was not providing it and I didn't have time to go down that rabbit hole.